pkg/flavor/kubernetes/nfs.go

```diff
@@ -60,7 +58,7 @@ type NFSSpec struct {
 }
 
 // CreateNFSVolume creates nfs volume abstracting underlying nfs pvc, deployment and service
-func (flavor *Flavor) CreateNFSVolume(pvName string, reqVolSize int64, parameters map[string]string, volumeContentSource *csi.VolumeContentSource) (nfsVolume *csi.Volume, rollback bool, err error) {
+func (flavor *Flavor) CreateNFSVolume(chapiDriver chapi.Driver, pvName string, reqVolSize int64, parameters map[string]string, volumeContentSource *csi.VolumeContentSource) (nfsVolume *csi.Volume, rollback bool, err error) {
```

Comment:
Rather than passing the chapi Driver around in different functions, can we instantiate an instance of it on the flavor when it get initialized?  Or give it the reference from the driver?

Reply:
done..thanks.